Makefile

```diff
@@ -191,10 +193,10 @@ deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in
 	kubectl apply -f $(CRD_MACHINE_CONFIG_POOL_URL)
 	kubectl apply -f $(CRD_KUBELET_CONFIG_URL)
 	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
-	$(KUSTOMIZE) build config/default | kubectl apply -f -
+	$(KUSTOMIZE) build $(KUSTOMIZE_DEPLOY_DIR)
```

Comment:
ah, the uncareful reviewer missed this bug!

Reply:
the uncareful author forgot the E2E GH action lane was disabled!